contracts/HATVaults.sol

```diff
@@ -340,7 +340,7 @@ contract  HATVaults is Governable, HATMaster {
      * @dev setRewardMultipliers - called by hats governance to set reward multipliers
      * @param _rewardMultipliers reward multipliers
     */
-    function setRewardMultipliers(uint256[25] memory _rewardMultipliers) external onlyGovernance {
+    function setRewardMultipliers(uint256[24] memory _rewardMultipliers) external onlyGovernance {
```

Comment:
Ok this is smart and will work, but it a bit implicit. I think it is a better pattern to use to make `rewardMultipliers` an array of 24 items, and change the logic in `getMultiplier`, wdyt?

Reply:
Yes that'd be the full solution I guess. Changed it now.